code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -184,3 +184,31 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/Madvilliansmask
+	name = "the MadVillian's mask"
+	desc = "A metal mask formerly owned by the Madvillian."
+	icon_state = "madvilliansmask"
+	inhand_icon_state = "madvilliansmask"
+	w_class = WEIGHT_CLASS_SMALL
+	modifies_speech = TRUE
+
+/obj/item/clothing/mask/Madvilliansmask/dropped(mob/M)
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
+
+/obj/item/clothing/mask/Madvilliansmask/handle_speech(datum/source, list/speech_args)
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message[1] != "*")
+		message = " [message]"
+		var/list/DOOMspeak_words = strings("DOOMspeak.json", "DOOMspeak")
+		for(var/key in DOOMspeak_words)
+			var/value = DOOMspeak_words[key]
```

Comment:
```suggestion
		var/list/doomspeak_words = strings("DOOMspeak.json", "DOOMspeak")
		for(var/key in doomspeak_words)
			var/value = doomspeak_words[key]
```

Reply:
It doesn't look like there's DOOMspeak.json anywhere though, this doesn't make sense.

Reply:
Yeah I forgot that I lifted It from spurdo code which is shitcode

Reply:
Doomspeak is in strings

Reply:
It's not included in your files changed, is this already in the code...?

Reply:
It's under commits

Reply:
Wait no Nevermind I must've added in in another branch by accident